extensions/extensions-docker/aissemble-fastapi/src/main/resources/docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Script for creating base FastAPI Docker image
 
-FROM python:3.11
+FROM docker.io/python:3.11
 
```

Comment:
i: explicitly stating the registry so fabric8 knows to authenticate to docker.io